backend/internal/config/config.go

```diff
@@ -9,3 +9,8 @@ const (
 	AttributeTeam   = "team"
 	AttributeAction = "action"
 )
+
+var (
```

Comment:
Should it not be const?

Reply:
Go doesn’t support const arrays or slices.

Reply:
Arrays can't be const.